recipes/pyqg/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pyqg" %}
+{% set version = "0.3.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pyqg/pyqg/archive/v{{ version }}.tar.gz
```

Comment:
There are a few issues with the tarball. I'll send a PR later to fix them but for now let's use the GH source b/c it has all the files we need.